requirements/thumbor.txt

```diff
@@ -0,0 +1,24 @@
+Pillow==3.4.2
+argparse==1.2.1
+backports-abc==0.4
+backports.ssl-match-hostname==3.5.0.1
+botocore==1.2.0
+certifi==2016.9.26
+derpconf==0.8.1
+docutils==0.12
+futures==3.0.5
+jmespath==0.7.1
+libthumbor==1.3.2
+pexif==0.15
+pycrypto==2.6.1
+pycurl==7.43.0
+python-dateutil==2.6.0
+pytz==2016.7
+singledispatch==3.4.0.3
+six==1.10.0
+statsd==3.2.1
+tc-aws==6.0.3
+thumbor==6.1.5
+tornado==4.4.2
+tornado-botocore==1.0.2
+typing
```

Comment:
I think it'd be nice to use have a `thumbor_locked.txt` that is generated from a basic `thumbor.txt` dependency, so that we don't have to maintain this list of dependency versions by hand.

See `tools/update-locked-requirements` for how we do this for other things.